worksheet_C/README.md

```diff
@@ -0,0 +1,9 @@
+- (a) The algorithm checks list i and j for duplicates, if a duplicate is found it shows the word true, if not show false
+- (b) For each item in list i it is being compared to every item before and including that same point in list j, e.g. list item 2 in i is being compared to list item 0,1,2 in j, written as n*n or n^2
+- (c) If line 4 is changed it still compares every item in list i and j for duplicates
+- (d) The algrorithm runs twice as fast because instead of comparing one item in list i against every item of list j, it only compares list numbers j below the list number i e.g. if the list number in i is 2 it only compares list number in j before 2
+- (e) I think the algorithm cannot be counted as quadratic anymore as the numbers are no longer being copared to its equivalent in list j therefore the equation is no longer n^2
```

Comment:
As explained in class, the algorithm is still quadratic -- the number of comparisons is n(n-1)/2 which simplifies to O(n^2).